pkg/gpsd/gpsd.go

```diff
@@ -0,0 +1,317 @@
+package gpsd
```

Comment:
This is an external package?  Can this be vendored in instead?

Reply:
no i have modified to add  TOFF and other report in my updated PR .yet t push it

Reply:
and they are outdated. the struct values are in nano sec  not micro sec

Reply:
vendored via https://github.com/aneeshkp/go-gpsd